Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,25 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Object Removal Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		if (m_LocalCoreStartupParameter.update)
+		{
+			// Set lock so codes can be enabled/disabled in game without crashes.
+			m_LocalCoreStartupParameter.done = false;
+			for (int current_object_removal_code = 0; current_object_removal_code < m_LocalCoreStartupParameter.num_object_removal_codes; current_object_removal_code++)
```

Comment:
As m_LocalCoreStartupParameter.object_removal_codes is a std::vector, you can just iterate in the much nicer c++11 way:
for (const SkipEntry& entry : m_LocalCoreStartupParameter.object_removal_codes)
if (!memcmp(src.GetPointer(), entry.data(), entry.size()))
return size;

So there is no need to also store the size of this buffers.